### PR TITLE
ui(fix): ENTESB-14064 - api connector details

### PR DESCRIPTION
**Pull request by kahboom**
Fixes issues with API Connector Details. adds some tests, i18n, and stories.

Quite a big PR but there were lots of changes that were difficult to separate into another PR. Also there were a few files moved around and renamed.

[ENTESB-14064](https://issues.redhat.com/browse/ENTESB-14064)

api-connector-detail-01

### CODE REVIEW

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/ApiConnectorReview.tsx [hunk 1]
```diff
@@ -37,113 +38,115 @@ export class ApiConnectorReview extends React.Component<
...
-                      <Text component={TextVariants.p} key={index}>
-                        {index + 1}. {errorMsg}
-                      </Text>
-                    )
-                  )
-                : null}
-
-              {/* warning messages */}
-              {this.props.i18nWarningsHeading && this.props.warningMessages && (
+                <Container>
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/ApiConnectorReview.tsx:60]: I think we want to try avoiding `Container`, no?  I thought this was mostly to work around integrating pf4 things into pf3 cards (@seanforyou23 correct me if I'm wrong), might be work fine to just remove this.

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailBody.tsx [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  };
+
+  const onSubmit = () => {
+    setIsEditing(false);
+    handleSubmit(configured);
+  };
+
+  return (
+    <PageSection>
+      <Container>
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailBody.tsx:104]: Same here, I don't think this is needed.

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailConfig.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  i18nLabelBaseUrl,
+  i18nLabelDescription,
+  i18nLabelHost,
+  i18nLabelName,
+  properties,
+}) => {
+  return (
+    <>
+      <TextContent data-testid={'api-connector-detail-config'}>
+        <Container>
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailConfig.tsx:35]: Same here.  I just peeked and yeah, this component just wraps it's content in a div with the `container-fluid` class, I don't think that's part of patternfly anymore.

**Comment by kahboom** [comment]
@gashcrumb - awesome, thanks! I just removed it now

File: app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailBody.tsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+  const onSubmit = () => {
+    setIsEditing(false);
+    handleSubmit(configured);
+  };
+
+  return (
+    <PageSection>
+      <div
+        className={'row row-cards-pf'}
```
Comment (gashcrumb) [inline app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailBody.tsx:105]: I think it'd be worth replacing this div with a layout component also.  I don't recall that `row` and `row-cards-pf` are used anymore.

Comment (gashcrumb) [reply app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailBody.tsx:105]: I think also if we can prefer the react components to using html + classnames we'll be better off.

Comment (kahboom) [reply app/ui-react/packages/ui/src/Customization/apiClientConnectors/detail/ApiConnectorDetailBody.tsx]: Turns out we didn't even need to add anything it after removing the `div`, just removed it and moved the `data-testid` over to the `PageSection` component

**Comment by gashcrumb** [comment]
retest this please

**Pull request merged**